src/InlineMethod.Fody/ModuleWeaver.cs

```diff
@@ -74,8 +74,8 @@ public override void Execute()
                 var attr = GetInlineAttribute(method);
                 if (attr != null)
                 {
-                    var remove = (bool)attr.ConstructorArguments.Single().Value;
-                    if (remove && method.IsPrivate)
+                    var behavior = (int)attr.ConstructorArguments.Single().Value;
+                    if (behavior > 0 && (behavior == 2 || method.IsPrivate))
```

Comment:
I think it's better to add a comment explaining what is 0, 2 and so on
Also I think it's better not to use greater than or less than comparison with enum values but to use equals and not equals.

Reply:
Removed raw enum value usage